<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResWgt</class>
 <widget class="QWidget" name="SearchResWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>61</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>61</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SearchResWgt {
	background-color: rgb(203,203,203);
	border-radius: 15px;
}

QLabel {
	font: 25 10pt '微软雅黑 Light';
	color: black;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="UsernameLb">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="UseridLb">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="AddSearchResPb">
     <property name="styleSheet">
      <string notr="true">QPushButton#AddSearchResPb {
    background-color: transparent;
    border-radius: 15px;
	font: 25 14pt '微软雅黑 Light';
	color: black;
}

    QPushButton#AddSearchResPb:hover {
        background-color: rgb(170,168,159);
    }

    QPushButton#AddSearchResPb:pressed {
        background-color: rgb(115,110,95);
    }</string>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
